common/lib/capa/capa/responsetypes.py

```diff
@@ -248,7 +248,9 @@ def render_html(self, renderer, response_msg=''):
           - response_msg: a message displayed at the end of the Response
         """
         # render ourself as a <span> + our content
-        tree = etree.Element('span')
+        tree = etree.Element('section')
+        tree.set('class', 'wrapper-problem-response')
+        tree.set('tabindex', '-1')
```

Comment:
@cptvitamin ^^ What do you think about this? Should we also add question label text to `aria-label` ?